src/Versity.Products/External/Presentation/Extensions/ProgramExtensions.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System.Reflection;
+using System.Text;
+using Application;
+using Infrastructure;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.DataProtection.AuthenticatedEncryption;
+using Microsoft.AspNetCore.DataProtection.AuthenticatedEncryption.ConfigurationModel;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.OpenApi.Models;
+using Serilog;
+using Serilog.Exceptions;
+using Serilog.Sinks.Elasticsearch;
+
+namespace Presentation.Extensions;
+
+public static class ProgramExtensions
+{
+    public static WebApplicationBuilder ConfigureBuilder(this WebApplicationBuilder builder)
+    {
+        builder.Services.AddDbContext(builder.Configuration);
+        builder.Services.AddRepositories();
+        builder.Services.AddApplication();
+        builder.Services.AddJwtAuthentication(builder.Configuration);
+        builder.Services.AddControllers();
+        builder.Services.AddEndpointsApiExplorer();
+        builder.Services.AddSwaggerGen(options =>
+        {
+            options.AddSecurityDefinition("oauth2", new OpenApiSecurityScheme()
+            {
+                In = ParameterLocation.Header,
+                Name = "Authorization",
+                Type = SecuritySchemeType.ApiKey
+            });
+        });
+
+        builder.Services.AddCors(options => options.AddPolicy("AllowAll", policy =>
+        {
+            policy.AllowAnyHeader();
+            policy.AllowAnyMethod();
+            policy.AllowAnyOrigin();
+        }));
+        
+        builder.Host.UseSerilog((context, configuration) =>
```

Comment:
Extension method

Reply:
I didn't quite understand what you meant here, but I put the method data in a [separate class and file](https://github.com/jejikeh/versity/pull/17/files#diff-6e8fd5248f660e8a2b4f8716f6d87f1b8d71a90379d24438ed5df6f0f9f6c06a)